tests/integration/src/all_data_types.rs

```diff
@@ -0,0 +1,174 @@
+use std::time::Duration;
+
+use anyhow::{Context, Result};
+use async_std::task;
+use fluvio_model_sql::{Operation, Upsert};
+use log::{debug, info};
+use serde::Deserialize;
+
+use crate::utils::{
+    self, ctx::TestContext, generate_records, new_config_path, produce_to_fluvio,
+    read_from_postgres,
+};
+
+pub async fn test_postgres_all_data_types(ctx: &mut TestContext) -> Result<()> {
```

Comment:
since this is fixture, no need to return result

Reply:
Sure, the intention it was handle erros with a teardown. But make sense, as we are using assets that panics does not make sense handle as Result. I removed them. 

Maybe we should try to recovery panics on main.rs to handle teardown.